superset-frontend/src/features/databases/DatabaseModal/index.tsx

```diff
@@ -1656,6 +1666,16 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
             value: target.value,
           })
         }
+        onEncryptedExtraInputChange={({
+          target,
+        }: {
+          target: HTMLInputElement;
+        }) =>
+          onChange(ActionType.EncryptedExtraInputChange, {
```

Comment:
This is super helpful!

Reply:
Yeah, I was surprised it didn't exist yet!